.github/pull_request_template.md

```diff
@@ -1,12 +0,0 @@
-# Description
```

Comment:
Why remove this file?

Reply:
It was already merged in this PR https://github.com/charmed-osm/srs-enb-ue-operator/pull/7

Reply:
Yes, it will 🤦🏼‍♂️,  reverted.